website/src/content/docs/getting-started/basics.mdx

```diff
@@ -0,0 +1,322 @@
+
+Let's test it out by doing the following:
+
+- Select the line we just changed with <kbd>x</kbd>.
+
+<T.SelectedLineWeJustChanged />
+
+- <kbd>d</kbd> to delete this line.
+
+<T.SelectedLineDeleted />
+
+- Spam <kbd>p</kbd> a few times to create some duplicates.
+
+<T.SelectedLineDuplicated />
+
+Now let's remove all of our previous `Hello helix!` by doing the following for each `Hello helix!` line:
+
+- Select the line with <kbd>x</kbd>.
+- <kbd>d</kbd> to delete it.
+
+Now we have something like this:
+
+<T.HelloHelixLinesRemoved />
+
+### Undo and Redo
+
+What if we made a mistake, and want to go back? The <kbd>u</kbd> command will _**u**ndo_ our most recent action. It's similar to `Ctrl-z` in other editors.
+
+Try pressing down <kbd>u</kbd> a few times to get to our previous state, before we made all those modifications:
+
+<T.PressedUAFewTimes />
+
+<kbd>U</kbd> is similar to <kbd>Ctrl</kbd> + <kbd>Z</kbd> in other editors. It will _undo the last undo_. It's the inverse of <kbd>u</kbd>.
+
+Press <kbd>U</kbd> until we get back to our most recent state:
+
+<T.BackToMostRecentState />
+
+### Checkpoint
+
+Feel free to make modifications to your file using the commands we have learned so far:
+
+- <kbd>h</kbd>, <kbd>j</kbd>, <kbd>k</kbd> and <kbd>l</kbd> moves 1 character left, down, up and right.
+- <kbd>i</kbd> enters Insert mode.
+- <kbd>esc</kbd> enters Normal mode.
+- <kbd>x</kbd> selects the entire line.
+- <kbd>y</kbd> yanks the selection.
+- <kbd>p</kbd> pastes the recently copied selection.
+- <kbd>e</kbd> moves to the end of the current word.
+- <kbd>b</kbd> moves to the beginning of the current word.
+- <kbd>;</kbd> removes the extra selection.
+- <kbd>d</kbd> deletes the current selection, without exiting Normal mode.
+- <kbd>c</kbd> changes the current selection, by deleting it and entering Insert mode.
+- <kbd>u</kbd> will undo the most recent change.
+- <kbd>U</kbd> will undo the most recent undo.
+
+Once you are happy with your modifications, enter Normal mode and type <kbd>:</kbd>.
+
+<kbd>:</kbd> enters _command mode_, which has commands you type out.
+
+- `:w` will _**w**rite_ the current file.
+- `:q` will _**q**uit_ the current file.
+- `:q!` will quit without saving.
+- `:wq` will both write and quit.
+
+<T.WriteQuitAll />
+
+## More Commands
+
+Let's try out more Helix commands! Open the file again with `hx file.txt`. Select the entire file by pressing <kbd>%</kbd>
+
+<T.FullFileSelected />
+
+Now, delete the selection with <kbd>d</kbd>.
+
+<T.FullFileDeleted />
+
+### Registers
+
+Helix has a concept called _registers_, which is like having many clipboards at once (hence the _"a clipboard"_). To interact with them, prefix yank and delete commands with a <kbd>"</kbd> and then the name of the register.
+
+For example, the contents of the system clipboard are stored inside the `+` register.
+
+:::note
+
+You may need a clipboard provider like [xclip](https://github.com/astrand/xclip) to use the system clipboard.
```

Comment:
```suggestion
You may need a clipboard provider like [xclip](https://github.com/astrand/xclip) to use the system clipboard.
```

I would specify this is a linux (also macOS ?) only requirement. On Windows, helix already binds to the system clipboard automatically.

Reply:
Yes, that's a valid point!